README.md

```diff
@@ -342,11 +357,35 @@ with only the `:protoc-gen-kotlin` changes, and then unstash the `:runtime` chan
 #### Conformance Tests
 
 To run conformance tests, the [conformance-test-runner](https://github.com/google/protobuf/tree/master/conformance) must
-be built (does not work on Windows). Then both the JS and JVM projects must be built via:
+be built (does not work on Windows).
+
+    $ git clone git@github.com:protocolbuffers/protobuf.git
+    $ cd protobuf
+    $ git checkout 3.10.x # Or another release branch
+    $ ./autogen.sh
+    $ cd conformance
+    $ make
+
+You should now have a `conformance-test-runner` available in this directory.  Test it by running `./conformance-test-runner --help`
+
+Set the `CONF_TEST_PATH` environment variable (used to run the tests below) with:
+
+    $ export CONF_TEST_PATH="$(pwd)/conformance-test-runner"
+
+Now, back in `pb-and-k`, build both the JS and JVM projects via:
+
+    $ ./gradlew :conformance:conformance-js:assemble
+    $ ./gradlew :conformance:conformance-jvm:installDist
+
+Bring in javascript dependencies:
+
+    $ cd conformance/conformance-js
+    $ yarn
+
+You are now ready to run the conformance tests.  Make sure `CONF_TEST_PATH` environment variable is set to `path/to/protobuf/conformance/conformance-test-runner` (see above).
+
+Then, from the root directory:
 
-    path/to/gradle :conformance:conformance-js:assemble
-    path/to/gradle :conformance:conformance-jvm:installDist
+    $ ./conformance/test-conformance.sh
```

Comment:
📌 Once we get the conformance tests passing, it'd be awesome to add a CircleCI job that runs them on every commit.

Reply:
Yes, definitely... they have a flag `--enforce-recommended` that will also require all the recommended test cases pass